lib/configure.js

```diff
@@ -81,11 +84,31 @@ function configure (gyp, argv, callback) {
         findVisualStudio(release.semver, gyp.opts.msvs_version,
           createConfigFile)
       } else {
+        createPythonSymlink()
         createConfigFile()
       }
     })
   }
 
+  function createPythonSymlink () {
+    const symlinkDestination = path.join(buildBinsDir, 'python3')
+
+    log.verbose('python symlink', `creating symlink to "${python}" at "${symlinkDestination}"`)
```

Comment:
```suggestion
    log.verbose('python symlink', `creating symlink to "${python3}" at "${symlinkDestination}"`)
```

Reply:
repeat below?

Reply:
In this string interpolation, `python` is a JS variable holding the path on disk to the Python binary chosen by `node-gyp`'s own `find-python.js`.

https://github.com/nodejs/node-gyp/blob/v9.0.0/lib/configure.js#L29

This variable's name doesn't functionally matter -- changing it would be only a cosmetic change, strictly for readers of this repo's JS code. We could change this variable in this file to be named `python3`, but would not change what is being logged. What is actually being logged (path to a usable Python binary) is determined by the logic in `find-python.js`.